_episodes/09-tdd.md

```diff
@@ -32,42 +37,45 @@ implemented for your current needs.
 > features. You do not spend any time implementing options and features on the off
 > chance that they will prove helpful later. You get what you need when you need it,
 > and no more. TDD is a very powerful idea, though it can be hard to follow religiously.
->
+{: .callout}
 
 The most important takeaway from test-driven development is that the moment
 you start writing code, you should be considering how to test that code. The
 tests should be written and presented in tandem with the implementation. **Testing
 is too important to be an afterthought.**
 
-> ## You Do You {.callout}
+> ## Who is using it?
```

Comment:
This was a nod to pop culture. The heading change doesn't make any sense. The paragraph is not at all about "who is using it
 Please give a compelling reason for this change or revert it.